lib/appcontroller_client.py

```diff
@@ -370,3 +379,150 @@ def set_deployment_id(self, deployment_id):
     return self.run_with_timeout(10, 'Set deployment ID request timed out.',
       self.DEFAULT_NUM_RETRIES, self.server.set_deployment_id, self.secret,
       deployment_id)
+
+
+  def get_all_stats(self):
+    """ Queries the AppController for all the stats.
+
+    Returns:
+      All of the application's stats from the AppController
+    """
+    #TODO: Check if this timeout increased to 30 is ok because this request takes some time.
+    return self.run_with_timeout(30, 'Get all JSON stats request timed out.',
```

Comment:
Please refrain from using hardcoded values. 
If the rest of this file has them they should be turned into constants; especially if they're used in multiple places.